packages/service-integration/tests/data/metadata.yml

```diff
@@ -1,9 +1,11 @@
-name: oSparc Python Runner
+name: Sim4Life Python Runner
 key: simcore/services/dynamic/osparc-python-runner
 type: computational
 integration-version: 1.0.0
 version: 1.1.0
-description: oSparc Python Runner
+version_display: "Sim4Life Release V7.2"
+release_date: "2024-05-30"
```

Comment:
and what if these 2 new fields are missing. Have we got a test for that? Or some test data for that as well? (thinking about the older versions)

Reply:
My question was more around safety: what if someone in the future changes this and make them required will that be detected as well in the tests?